### DISCUSSION TITLE
Is it better to use host browser or browser from within another VM like Ubuntu?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#5649] [Unsupported Versions] [answered]
Hi, 
   I am using windows 10 from my laptop as my host for my VMs. I have created 3 SO VMs:
   SO 2.3.70-WAZUH 17 eval
   SO 2.3.70-WAZUH 17 import
   SO 2.3.70-WAZUH 17 standalone
   I have heard that you want to use a browser from within a VM like ubuntu to access the SOC? I guess this is because of sniffing traffic that might affect your laptop host if you use the host browser? 
   thanks for your suggestions and advice

**Comment by clairmont32** [comment, marked as answer]
IMO, it kind of depends on the tasks you're going to be performing. If you intend to carve files out of PCAPs, it's best to do an install of the SO Analyst variant (`./so-install-analyst`) so you have a dedicated VM with a bunch of additional analysis tools. The other, smaller, added benefit is if something with a Linux malware accidentally takes place you can throw the VM away.

If you _don't_ plan on doing file extracts from PCAPs and only review alerts and logs, you shouldnt need another VM and can get by with just your Win 10 host browser.
